pipelines/tikv/migration/latest/pull_unit_test.groovy

```diff
@@ -0,0 +1,93 @@
+        timeout(time: 40, unit: 'MINUTES')
+        parallelsAlwaysFailFast()
+        skipDefaultCheckout()
+    }
+    stages {
+        stage('Debug info') {
+            steps {
+                sh label: 'Debug info', script: """
+                    printenv
+                    echo "-------------------------"
+                    go env
+                    echo "-------------------------"
+                    echo "debug command: kubectl -n ${K8S_NAMESPACE} exec -ti ${NODE_NAME} bash"
+                """
+                container(name: 'net-tool') {
+                    sh 'dig github.com'
+                }
+            }
+        }
+        stage('Checkout') {
+            options { timeout(time: 5, unit: 'MINUTES') }
+            steps {
+                dir("migration") {
+                    cache(path: "./", filter: '**/*', key: "git/tikv/migration/rev-${ghprbActualCommit}", restoreKeys: ['git/tikv/migration/rev-']) {
+                        retry(2) {
+                            checkout(
+                                changelog: false,
+                                poll: false,
+                                scm: [
+                                    $class: 'GitSCM', branches: [[name: ghprbActualCommit]],
+                                    doGenerateSubmoduleConfigurations: false,
+                                    extensions: [
+                                        [$class: 'PruneStaleBranch'],
+                                        [$class: 'CleanBeforeCheckout'],
+                                        [$class: 'CloneOption', timeout: 15],
+                                    ],
+                                    submoduleCfg: [],
+                                    userRemoteConfigs: [[
+                                        refspec: "+refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/*",
+                                        url: "https://github.com/${GIT_FULL_REPO_NAME}.git",
+                                    ]],
+                                ]
+                            )
+                        }
+                    }
+                }
+            }
+        }
+        stage('Unit Tests') {             
+            stages {
+                stage("Test") {
+                    options { timeout(time: 25, unit: 'MINUTES') }
+                    environment {TIKV_MIGRATION_CODECOV_TOKEN = credentials('codecov-token-tikv-migration')}
+                    steps {
+                        dir('migration') {
+                            sh """
+                                cd cdc/
+                                make unit_test_in_verify_ci
+                            """
+                        }
+                    }
+                    post{
+                        always {
+                            junit(testResults: "**/cdc-junit-report.xml")
+                        }
+                    }
+                }
+            }         
+        }
+    }
```

Comment:
should it report result in post of pipeline?

Reply:
No need now